tests/image/image.cpp

```diff
@@ -83,6 +84,7 @@ class ImageTestCase : public CppUnit::TestCase
         CPPUNIT_TEST( DibPadding );
         CPPUNIT_TEST( BMPFlippingAndRLECompression );
         CPPUNIT_TEST( ScaleCompare );
+    CPPUNIT_TEST( CreateBitmapFromCursor );
```

Comment:
Minor but indentation is wrong
```suggestion
        CPPUNIT_TEST( CreateBitmapFromCursor );
```

Reply:
@vadz,
I made this in Xcode which treats spaces as hard-TABs.

Will fix.

Thx.

Reply:
@oneeyeman1 for Xcode: go into Xcode->Preferences, Section "Text Editing", Subsection "Indentation", set

Prefer Indent Using: to Spaces
Tab Key: Intents in leading whitespaces
and both to 4 spaces

HTH